index.html

```diff
@@ -95,6 +95,14 @@
 
       <div class="container marketing p-md-5">
 
+
+        <div class="alert alert-warning alert-dismissible fade show" role="alert">
```

Comment:
I also received some styling suggestions, which we are free to take or leave

```suggestion
        <div class="alert alert-warning alert-dismissible fade show" role="alert" style="text-align: center;font-size: 1.1em;max-width: 720px;margin-left: auto;margin-right: auto;">
```